syspurpose/src/syspurpose/cli.py

```diff
@@ -44,7 +44,7 @@ def add_command(args, syspurposestore):
     for value in args.values:
         if syspurposestore.add(args.prop_name, value):
             any_prop_added = True
-            print(_("Added {value} to {prop_name}.").format(
+            print(_("Added \"{value}\" to {prop_name}.").format(
                 value=make_utf8(value), prop_name=make_utf8(args.prop_name)))
         else:
             print(_("Not adding value {value} to {prop_name}; it already exists.").format(
```

Comment:
I would surround `{value}` with quotes on this line too. I didn't find any other missing quotas.